cmd/crc/cmd/stop.go

```diff
@@ -22,15 +26,46 @@ var stopCmd = &cobra.Command{
 	},
 }
 
-func runStop(arguments []string) {
+var (
+	force bool
+)
 
+func runStop(arguments []string) {
 	stopConfig := machine.StopConfig{
 		Name: constants.DefaultName,
 	}
 
+	killConfig := machine.KillConfig{
+		Name: constants.DefaultName,
+	}
+
+	if force {
+		killVM(killConfig)
+		errors.Exit(0)
+	}
+
 	commandResult, err := machine.Stop(stopConfig)
+	if err != nil {
+		// Here we are catching the error "VM Failed to gracefully shutdown, try the kill command"
+		// from the driver side and if it exist then we ask user input for power off operation.
+		if strings.Contains(err.Error(), "kill") {
```

Comment:
:+1:

Reply:
do we need to do this? the result is a `false`. can't we use this instead?
t he state is running and returned false on stop.,,

besides, this error message needs to be modified in the driver, which means tis check is fragile.

Reply:
Problem is the state, if your VM is in `stop` or `delete` state then error will be returned and user have to prompted for it which doesn't make sense :(

Reply:
This got changed by additionally checking for state == running.